components/common/ItemTransferModal.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <UserCartModal
+    ref="userCartModal"
+    mode="transfer"
+    :title="$t('transaction.transferNft', items.length)"
+    :signing-title="$t('transaction.transferingNft', items.length)"
+    :get-action="getAction"
+    :label="label"
+    :disabled="disabled"
+    :loading="loading"
+    @reset="reset"
+  >
+    <template #body>
+      <hr class="my-4">
+
+      <h2 class="mb-2 font-bold text-text-color capitalize">
+        {{ $t('transaction.transferTo') }}
+      </h2>
+
+      <AddressInput
```

Comment:
missing `strict` prop? required in this component, or make it optional

Reply:
f30179d6a46163c8160a640d2eb786b799c53971 made it optional . already had a default value